Visage.FrontEnd/Visage.FrontEnd.Shared/Styles/input.css

```
@@ -1,3 +1,23 @@
-@tailwind utilities;
+﻿@import "tailwindcss";
+@plugin "@daisyui" {
+    themes: light --default, dark --prefersdark;
+    root: ":root";
+    include:;
+    exclude:;
+    prefix:;
+    logs: true;
+}
+@plugin "daisyui/theme" {
+    name: "light";
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)